ArjunaJTS/orbportability/classes/com/arjuna/orbportability/ORBType.java

```diff
@@ -64,11 +64,9 @@ static int getORBEnum(String name)
 
 
     public static final int JAVAIDL = 1;
-    public static final int IBMORB = 5;
```

Comment:
@jmfinelli - given this is a public API change, it should require at least a minor version bump for the release

Reply:
I will mention this in the JIRA. Thanks Tom